cloudmailer/cloudmailer.py

```diff
@@ -132,7 +132,7 @@ def getCredentials(self):
         for key in cred:
             if not cred[key]:
                 print('Credentials not loaded to environment: did you load the rc file?')
-                exit(1)
+                sys.exit(1)
```

Comment:
Same as #123

Reply:
Yeah, I can revert all the `sys.exit(1)` , if you prefere.